source/_integrations/qvr_pro.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: "QVR Pro"
+description: "Instructions on how to integrate QVR Pro within Home Assistant."
+logo: qvr_pro.png
+ha_category:
+  - Camera
+ha_release: "0.106"
+---
+
+[QVR Pro](https://www.qnap.com/solution/qvr-pro-official) allows you to create 
+an independent and expandable surveillance environment on your QNAP NAS.  The 
+`qvr_pro` component allows you to view your QVR Pro channels in Home Assistant.
+
+Currently only cameras are supported by this integration.
+
+## Configuration
+
+To enable QVR Pro integration, add the following to your
+`configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+qvr_pro:
+  host: 192.168.1.102
+  username: pyqvruser
+  password: password
+  exclude_channels: 2,5,6
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.